.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->
+
+### Details
+<!-- Explanation of the change or anything fishy that is going on -->
+
+### Related Issues
+<!-- Please replace GH_LINK with the link to the GitHub issue this Pull Request is related to -->
+GH_LINK
+
+### Automated Tests
+<!---
+Most changes to Onyx should have accompanying tests. Describe the tests you added or if no tests were added an explanation about why one was not needed.
+--->
```

Comment:
Let's change this to `### Manual Tests` please and remove the mention of Onyx

Reply:
Ah, good call. I didn't audit this at all.